Makefile

```diff
@@ -25,7 +25,7 @@ JS ?= samples/HelloWorld.js
 VARIANT ?= release
 # JerryScript options
 JERRY_BASE ?= $(ZJS_BASE)/deps/jerryscript
-EXT_JERRY_FLAGS ?= -DENABLE_ALL_IN_ONE=ON
+EXT_JERRY_FLAGS ?= -DENABLE_ALL_IN_ONE=ON -DFEATURE_PROFILE=$(ZJS_BASE)/zephyr.profile
```

Comment:
maybe another name like jerry_feature.profile to indicate that it's JrS-specific, the name suggests it is a zephyr config which is confusing.